app/src/main/java/com/isc/hermes/utils/offline/OfflineMapManager.java

```diff
@@ -0,0 +1,160 @@
+package com.isc.hermes.utils.offline;
+import android.content.Context;
+
+import com.isc.hermes.R;
+import com.mapbox.mapboxsdk.Mapbox;
+import com.mapbox.mapboxsdk.geometry.LatLngBounds;
+import com.mapbox.mapboxsdk.offline.OfflineManager;
+import com.mapbox.mapboxsdk.offline.OfflineRegion;
+import com.mapbox.mapboxsdk.offline.OfflineTilePyramidRegionDefinition;
+
+import org.json.JSONObject;
+
+import timber.log.Timber;
+
+/**
+ * This class is in charged for managing offline maps.
+ */
+public class OfflineMapManager {
+    private OfflineManager offlineManager;
```

Comment:
`OfflineMapManager` and `OfflineManager` are pretty much similar and is confusing

Reply:
This class contains several unused lines of code.